FrontEnd/Hopsy/src/screens/ProfileScreen.js

```diff
@@ -29,19 +29,19 @@ export default class ProfileScreen extends React.Component {
           username: GLOBAL.user,
         })
     })  
-    .then((response) => response.text())
-      .then((responseJson) => {
-        this.setState({
-          fullname: responseJson,
-        }, function() {
-        })
-      })
+    .then(response => {
+      this.setState({
+        fullname: JSON.parse(response),
```

Comment:
I would add a comment here explaining what is actually going on since we are only extracting a few bits of information from the json

Reply:
Explanatory comment added in https://github.com/nauticalmonkey/CSC308/commit/9273d5ca9b47ebdbb5da410f31ace01fe27bcb11